linux/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -249,7 +250,7 @@
         vm_get_fullname_timeout: 600
       when:
         - unattend_install_conf is defined
-        - unattend_install_conf | lower is not match('.*(minimal|server_without_gui|bclinux-for-euler).*')
+        - unattend_install_conf | lower is not match('.*(minimal|server_without_gui|bclinux-for-euler|fusionos).*')
```

Comment:
Doesn't FusionOS have built-in open-vm-tools?

Reply:
No built-in ovt in FusionOS-22/23 and no related packages in iso. Linkup network interface after autoinstall but no ip until reboot.